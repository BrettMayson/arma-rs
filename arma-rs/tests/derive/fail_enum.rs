use arma_rs_proc::{FromArma, IntoArma};

#[derive(IntoArma, FromArma)]
enum DeriveTest {}

fn main() {}
