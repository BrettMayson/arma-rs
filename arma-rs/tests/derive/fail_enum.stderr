error: Enums aren't supported
 --> tests/derive/fail_enum.rs:3:10
  |
3 | #[derive(IntoArma, FromArma)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `IntoArma` (in Nightly builds, run with -Z macro-backtrace for more info)

error: Enums aren't supported
 --> tests/derive/fail_enum.rs:3:20
  |
3 | #[derive(IntoArma, FromArma)]
  |                    ^^^^^^^^
  |
  = note: this error originates in the derive macro `FromArma` (in Nightly builds, run with -Z macro-backtrace for more info)
